libmscore/album.cpp

```diff
@@ -0,0 +1,248 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "album.h"
+#include "score.h"
+#include "xml.h"
+#include "musescoreCore.h"
+
+namespace Ms {
+
+//---------------------------------------------------------
+//---------------------------------------------------------
+//
+//   AlbumItem
+//
+//---------------------------------------------------------
+//---------------------------------------------------------
+
+AlbumItem::AlbumItem(Album* album)
+{
+    this->album = album;
```

Comment:
Make class variables prefixed with `m_`, as stated in the new [code guideline](https://github.com/musescore/Documentation/blob/master/WorkflowAndGuidelines/CodeGuidelines.md). Then we don't need `this->` here.

Reply:
Thank you, I'll take care of it